config/clusters/2i2c-uk/staging.values.yaml

```diff
@@ -43,5 +43,3 @@ jupyterhub:
           http://google.com/accounts/o8/id:
             username_derivation:
               username_claim: "email"
-            allowed_domains:
-              - "2i2c.org"
```

Comment:
Removing ourselves from a CILogonOAuthenticator `allowed_domains` is also fine, we are allowed by being added to `admin_users`.